source/gloperate/include/gloperate/rendering/CameraUtils.h

```diff
@@ -0,0 +1,41 @@
+
+#pragma once
+
+
+#include <glm/mat4x4.hpp>
+
+#include <gloperate/gloperate_api.h>
+
+
+namespace gloperate
+{
+
+/**
```

Comment:
Please put static utility functions inside a class (see for example GLContextUtils, or gloperate_qtquick::Utils). The Name of the class should match the name of the file.